articles/azure/azs-how-create-S2S-VPN-azure-to-azurestack.md

```diff
@@ -0,0 +1,485 @@
+
+   - **SKU** - Route-based VPN gateway types are offered in three SKUs: Basic, Standard, and High performance. You must select Standard or High performance if you are creating the network to coexist with an ExpressRoute gateway. You must select High performance SKU to enable active-active mode. You can find more information about SKUs here: [Azure Stack SKUs](https://docs.microsoft.com/en-gb/azure/vpn-gateway/vpn-gateway-about-skus-legacy).
+
+   - **Virtual Network** - This is the virtual network that you created earlier in Public Azure.
+
+   - **Public IP address** - The public IP address to assign to this virtual network gateway. Only dynamic public IP addresses are supported.
+
+       - Click **Create new**.
+
+       - In the **Public IP address name** field, enter a public IP address name.
+
+   - **Enable active-active mode** - The gateway SKU must be either **VpnGw1**, **VpnGw2**, **VpnGw3**, **VpnGw1AZ**, **VpnGw2AZ**, **VpnGw3Az** or HighPerformance (legacy SKU).
+
+   - **Configure BGP ASN** - BGP is the standard routing protocol commonly used on the internet to exchange routing information between two or more networks. BGP enables the Azure VPN Gateways and your on-premises VPN devices, called BGP peers or neighbours, to exchange "routes" that will inform both gateways on the availability and reachability for those prefixes to go through the gateways or routers involved. You should also make sure your on-premises VPN devices support BGP before you enable this feature.
+
+   > [!NOTE]
+   > The virtual network gateway will be created in the same resource group as the chosen virtual network.
+
+   ![Create new virtual network gateway](images/azs-public-browser-create-virtual-network-gateway.png)
+
+6. Click **Review + create**.
+
+7. Review the configuration and click **Create**.
+
+## Create the local network gateway in Public Azure
+
+The local network gateway refers to your on-premises network. The following steps outline how to create a local network gateway:
+
+1. In the favourites panel, select **Create a resource**.
+
+   ![Create a resource option in favourites panel](images/azs-public-browser-create-resource-button.png)
+
+2. In the **New** blade, search for `Local network gateway`.
+
+   ![Create new networking resource](images/azs-public-browser-new-local-network-gateway-search.png)
+
+3. In the **Local network gateway** blade, click **Create**.
+
+    ![Create button in local network gateway blade](images/azs-public-browser-new-local-network-gateway-create.png)
+
+4. In the **Create local network gateway** blade, enter the following information:
+
+   - **Name** - The name of the local network gateway.
+
+   - **IP address** - This is the public IP address of the VPN device that you want Azure to connect to. Specify a dummy IP address, as this will need to updated once you have generated the public IP with the connection.
```

Comment:
I'd rephrase this a little to:
This is the public IP address of the VPN device that you want Azure to connect to. Enter a dummy IP address for now as you will generate the public IP later.